src/Components/RightSidebar/RequestSignatureTab.vue

```diff
@@ -58,6 +58,11 @@
 				@click="validationFile()">
 				{{ t('libresign', 'Validate') }}
 			</NcButton>
+			<NcButton v-if="filesStore.canValidate()"
```

Comment:
I think that this option will be always visible and don't will be necessary a condition to display the button.